docs/compass/04-01-installation.md

```diff
@@ -1,78 +1,68 @@
+    | From the `master` branch 	| `master`          	| `master`                	|
+    | From the specific commit on the `master` branch 	| `master-{COMMIT_HASH}` 	| `master-34edf09a` 	|
+    | From the specific PR       	| `PR-{PR_NUMBER}`         	| `PR-1420`     	|
 
-This is a multi-tenant and multi-Runtime mode that provides a cluster with Compass and only the selected Kyma components that Compass uses. To enable this mode, create this ConfigMap and then perform the cluster Kyma installation using the
- [`installer-cr-cluster-compass.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-compass.yaml.tpl) configuration file:
-
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: compass-overrides
-  namespace: kyma-installer
-  labels:
-    installer: overrides
-    component: compass
-    kyma-project.io/installation: ""
-data:
-  # The parameter that enables the Compass gateway, as the default Kyma gateway is disabled in this installation mode
-  gateway.gateway.enabled: "true"
-  # The name of the currently used gateway
-  global.istio.gateway.name: compass-istio-gateway
-  # The Namespace of the currently used gateway
-  global.istio.gateway.namespace: compass-system
-  # The parameter that disables preconfiguration for the Compass Runtime Agent
-  global.agentPreconfiguration: "false"
-  # The Namespace with a Secret that contains a certificate for the Connector Service
-  global.connector.secrets.ca.namespace: compass-system
+    Once you decide on the installation option, use this command:
+    ```bash
+    kubectl apply -f https://storage.cloud.google.com/kyma-development-artifacts/compass/{INSTALLATION_OPTION}/compass-installer.yaml
+    ```
+​
+4. Check the installation progress. To do so, download the script that checks the progress of the installation:
+```bash
+wget https://storage.cloud.google.com/kyma-development-artifacts/compass/{INSTALLATION_OPTION}/is-installed.sh && chmod +x ./is-installed.sh
+```
+​
+Then, use the script to check the progress of the Compass installation:
+```bash
+./is-installed.sh
 ```
 
-### Kyma Runtime
+## Local installation
 
-This is a single-tenant mode that provides complete cluster Kyma installation with the Runtime Agent. To enable this mode, follow the cluster Kyma installation using the [`installer-cr-cluster-runtime.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-runtime.yaml.tpl) configuration file. The default [legacy mode](#architecture-application-connector-components-application-operator) used in Kyma does not support integration with Compass. For that reason, before you start the installation, apply the following ConfigMap which disables components used in the legacy mode, such as the Application Registry and the Connector Service:
+For local development, install Compass along with the minimal Kyma installation from the `master` branch. To do so, run this script:
```

Comment:
```suggestion
For local development, install Compass along the minimal Kyma installation from the `master` branch. To do so, run this script:
```
I'm wondering... `along with` or `along`? 🤔 
Does this script install both Compass and Kyma, or does it only install Compass and the sentence tells you that you need the minimal Kyma installed?

Reply:
The script installs both Compass and Kyma.